nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/blob/ColdStorageHelper.java

```diff
@@ -138,6 +204,29 @@ protected static String getContentBlobKey(Blob coldContent) {
         return key;
     }
 
+    /**
+     * Status about the cold storage content being retrieved or available for a given repository.
```

Comment:
I would remove "for a given repository" since the session is actually handled by `checkColdStorageContentAvailability`.